src/coprocessor/codec/mysql/decimal.rs

```diff
@@ -572,9 +580,13 @@ fn do_div_mod(
         let frac_cnt = cmp::max(lhs.frac_cnt, rhs.frac_cnt);
         Res::Ok(Decimal::new(0, frac_cnt, lhs.negative))
     } else {
-        frac_word_to = word_cnt!(l_frac_cnt + r_frac_cnt + frac_incr);
+        frac_word_to = word_cnt!(
+            l_frac_cnt
```

Comment:
How about use usize instead? Will it make life better?

Reply:
I think it's similar to use `xxx as usize` and `saturating_add` here?